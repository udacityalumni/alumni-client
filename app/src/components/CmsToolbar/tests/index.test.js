import CmsToolbar from '../index';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';

describe('<CmsToolbar />', () => {
  it('should render with default props', () => {
    const wrapper = shallow(
      <CmsToolbar
        onSave={e => e}
        canSubmit
        onPreview={e => e}
      />
    );
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
