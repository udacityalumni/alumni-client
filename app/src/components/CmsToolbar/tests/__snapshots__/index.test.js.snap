exports[`<CmsToolbar /> should render with default props 1`] = `
<Box
  align="center"
  announce={false}
  direction="column"
  focusable={true}
  full="horizontal"
  justify="center"
  pad="none"
  responsive={true}
  tag="div">
  <Columns
    full="horizontal"
    justify="center"
    maxCount={1}
    responsive={true}>
    <Box
      align="center"
      announce={false}
      colorIndex="light-2"
      direction="row"
      focusable={true}
      full="horizontal"
      justify="between"
      margin="small"
      pad="medium"
      responsive={true}
      tag="div">
      <Button
        label="Submit"
        onClick={[Function onSave]}
        type="button" />
      <Button
        label="Preview"
        onClick={[Function onPreview]}
        type="button" />
    </Box>
  </Columns>
</Box>
`;
